src/Controller/Component/PropertiesComponent.php

```diff
@@ -26,6 +27,23 @@
  */
 class PropertiesComponent extends Component
 {
+    /**
+     * Schema property types
+     */
+    public const CUSTOM_PROPERTY_TYPES = [
```

Comment:
We should read `property_types` via `GET /model/property_types` to include `core` and `custom` types.

It would be nice to see the JSON-SCHEMA once a property is selected (but we can to this later)